ExponeaSDK/ExponeaSDK/Classes/Push Notifications/PushNotificationSwizzler.swift

```diff
@@ -51,8 +51,10 @@ final class PushNotificationSwizzler {
     }
 
     func addAutomaticPushTracking() {
-        swizzleTokenRegistrationTracking()
-        swizzleNotificationReceived()
+        DispatchQueue.main.async {
+            self.swizzleTokenRegistrationTracking()
+            self.swizzleNotificationReceived()
+        }
```

Comment:
As we do not recommend using swizzling for push notifications (see the first comment), automaticPushNotificationTracking is set to false in our code and method addAutomaticPushTracking should not be even called. Therefore, this change also cannot be accepted. We would be modifying already deprecated functionality. 

If you have any problems with migrating from automaticPushNotificationTracking, please let us know, we tried to cover every important info in the documentation.